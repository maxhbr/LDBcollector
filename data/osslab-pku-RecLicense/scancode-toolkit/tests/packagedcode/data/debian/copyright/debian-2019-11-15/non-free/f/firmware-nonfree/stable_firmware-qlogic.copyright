Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
Upstream-Name: linux-firmware
Upstream-Contact: linux-firmware@kernel.org
Source: https://git.kernel.org/cgit/linux/kernel/git/firmware/linux-firmware.git

Files: qlogic/1040.bin, qlogic/1280.bin, qlogic/12160.bin
Copyright: 1995, 1996, 1997, 1998, 1999, 2000, QLogic, Inc.
License: Binary redistribution (QLogic 1)
 Redistribution and use in source and binary forms are permitted provided
 that the following conditions are met:
 1. Redistribution of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer.
 2. Redistribution in binary form must reproduce the above copyright
    notice, this list of conditions and the following disclaimer in the
    documentation and/or other materials provided with the distribution.
 3. The name of the author may not be used to endorse or promote products
    derived from this software without specific prior written permission
 .
 THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
 IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
 OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
 IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
 INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
 NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
 DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
 THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Files: sd7220.fw:
Copyright: 2007, 2008. QLogic Corporation
License: The OpenIB.org BSD license
 All rights reserved.
 .
 Redistribution and use in source and binary forms, with or
 without modification, are permitted provided that the following
 conditions are met:
 .
  - Redistributions of source code must retain the above
    copyright notice, this list of conditions and the following
    disclaimer.
 .
  - Redistributions in binary form must reproduce the above
    copyright notice, this list of conditions and the following
    disclaimer in the documentation and/or other materials
    provided with the distribution.
 .
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
 EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
 MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
 NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
 BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
 ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.

Files: ql2100_fw.bin, ql2200_fw.bin, ql2300_fw.bin, ql2322_fw.bin,
 ql2400_fw.bin, ql2500_fw.bin
Copyright: 2003-2006, QLogic Corporation
License: Binary redistribution (QLogic 2)
 You may redistribute the hardware specific firmware binary file
 under the following terms:
 .
	1. Redistribution of source code (only if applicable),
	   must retain the above copyright notice, this list of
	   conditions and the following disclaimer.
 .
	2. Redistribution in binary form must reproduce the above
	   copyright notice, this list of conditions and the
	   following disclaimer in the documentation and/or other
	   materials provided with the distribution.
 .
	3. The name of QLogic Corporation may not be used to
	   endorse or promote products derived from this software
	   without specific prior written permission
 .
 REGARDLESS OF WHAT LICENSING MECHANISM IS USED OR APPLICABLE,
 THIS PROGRAM IS PROVIDED BY QLOGIC CORPORATION "AS IS'' AND ANY
 EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
 IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
 PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHOR
 BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
 EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
 TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
 DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
 ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 POSSIBILITY OF SUCH DAMAGE.
 .
 USER ACKNOWLEDGES AND AGREES THAT USE OF THIS PROGRAM WILL NOT
 CREATE OR GIVE GROUNDS FOR A LICENSE BY IMPLICATION, ESTOPPEL, OR
 OTHERWISE IN ANY INTELLECTUAL PROPERTY RIGHTS (PATENT, COPYRIGHT,
 TRADE SECRET, MASK WORK, OR OTHER PROPRIETARY RIGHT) EMBODIED IN
 ANY OTHER QLOGIC HARDWARE OR SOFTWARE EITHER SOLELY OR IN
 COMBINATION WITH THIS PROGRAM.

Files: cbfw-*.bin, ct2fw-*.bin, ctfw-*.bin
Copyright: 2013-2014, Brocade Communications Systems, Inc.
 2014-2015, QLogic Corporation.
License: Binary redistribution (QLogic BR-series)
 Permission is hereby granted for the distribution of this firmware data
 in hexadecimal or equivalent format, provided this copyright notice is
 accompanying it.
 .
 QLogic grants permission to use and redistribute these firmware files
 for use with QLogic BR-series devices, but not as a part of the Linux
 kernel or in any other form which would require these files themselves
 to be covered by the terms of the GNU General Public License.
 These firmware files are distributed in the hope that they will be
 useful, but WITHOUT ANY WARRANTY; without even the implied warranty
 of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Files: qed/qed_init_values_zipped-*.bin
Copyright: 2015-2016, QLogic Corporation
License: Binary redistribution (QLogic 3)
 Permission is hereby granted for the distribution of this firmware data
 in hexadecimal or equivalent format, provided this copyright notice is
 accompanying it.
