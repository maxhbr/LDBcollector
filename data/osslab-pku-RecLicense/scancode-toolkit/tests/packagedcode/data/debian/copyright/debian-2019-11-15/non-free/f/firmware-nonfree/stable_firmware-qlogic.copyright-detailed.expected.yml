primary_license: None
declared_license:
  - Binary redistribution (QLogic 1)
  - The OpenIB.org BSD license
  - Binary redistribution (QLogic 2)
  - Binary redistribution (QLogic BR-series)
  - Binary redistribution (QLogic 3)
license_expression: bsd-new AND linux-openib AND qlogic-microcode AND (other-permissive AND
  proprietary-license) AND other-permissive
copyright: |
  1995, 1996, 1997, 1998, 1999, 2000, QLogic, Inc.
  2007, 2008. QLogic Corporation
  2003-2006, QLogic Corporation
  2013-2014, Brocade Communications Systems, Inc.
  2014-2015, QLogic Corporation.
  2015-2016, QLogic Corporation
matches:
  - score: '100.0'
    start_line: 9
    end_line: 28
    matcher: 1-hash
    rule_length: '199'
    matched_length: '199'
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: bsd-new_1070.RULE
    license_expression: bsd-new
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      Redistribution and use in source and binary forms are permitted provided
      that the following conditions are met:
      1. Redistribution of source code must retain the above copyright
         notice, this list of conditions and the following disclaimer.
      2. Redistribution in binary form must reproduce the above copyright
         notice, this list of conditions and the following disclaimer in the
         documentation and/or other materials provided with the distribution.
      3. The name of the author may not be used to endorse or promote products
         derived from this software without specific prior written permission

      THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR
      IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
      OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
      IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,
      INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
      NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
      DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
      THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
      (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
      THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  - score: '100.0'
    start_line: 35
    end_line: 55
    matcher: 2-aho
    rule_length: 143
    matched_length: 143
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: linux-openib.LICENSE
    license_expression: linux-openib
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      Redistribution and use in source and binary forms, with or
      without modification, are permitted provided that the following
      conditions are met:

       - Redistributions of source code must retain the above
         copyright notice, this list of conditions and the following
         disclaimer.

       - Redistributions in binary form must reproduce the above
         copyright notice, this list of conditions and the following
         disclaimer in the documentation and/or other materials
         provided with the distribution.

      THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
      EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
      MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
      NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
      BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
      ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
      CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
      SOFTWARE.
  - score: '100.0'
    start_line: 61
    end_line: 96
    matcher: 1-hash
    rule_length: 260
    matched_length: 260
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: qlogic-microcode.LICENSE
    license_expression: qlogic-microcode
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      You may redistribute the hardware specific firmware binary file
      under the following terms:

      1. Redistribution of source code (only if applicable),
      must retain the above copyright notice, this list of
      conditions and the following disclaimer.

      2. Redistribution in binary form must reproduce the above
      copyright notice, this list of conditions and the
      following disclaimer in the documentation and/or other
      materials provided with the distribution.

      3. The name of QLogic Corporation may not be used to
      endorse or promote products derived from this software
      without specific prior written permission

      REGARDLESS OF WHAT LICENSING MECHANISM IS USED OR APPLICABLE,
      THIS PROGRAM IS PROVIDED BY QLOGIC CORPORATION "AS IS'' AND ANY
      EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
      IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
      PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHOR
      BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
      EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
      TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
      DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
      ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
      OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
      OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
      POSSIBILITY OF SUCH DAMAGE.

      USER ACKNOWLEDGES AND AGREES THAT USE OF THIS PROGRAM WILL NOT
      CREATE OR GIVE GROUNDS FOR A LICENSE BY IMPLICATION, ESTOPPEL, OR
      OTHERWISE IN ANY INTELLECTUAL PROPERTY RIGHTS (PATENT, COPYRIGHT,
      TRADE SECRET, MASK WORK, OR OTHER PROPRIETARY RIGHT) EMBODIED IN
      ANY OTHER QLOGIC HARDWARE OR SOFTWARE EITHER SOLELY OR IN
      COMBINATION WITH THIS PROGRAM.
  - score: '100.0'
    start_line: 102
    end_line: 104
    matcher: 2-aho
    rule_length: 23
    matched_length: 23
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: other-permissive_66.RULE
    license_expression: other-permissive
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      Permission is hereby granted for the distribution of this firmware data
      in hexadecimal or equivalent format, provided this copyright notice is
      accompanying it.
  - score: '97.33'
    start_line: 106
    end_line: 112
    matcher: 3-seq
    rule_length: 73
    matched_length: 73
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: proprietary-license_310.RULE
    license_expression: proprietary-license
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      grants permission to use and redistribute these firmware files
      for use with [QLogic] BR-[series] devices, but not as a part of the Linux
      kernel or in any other form which would require these files themselves
      to be covered by the terms of the GNU General Public License.
      These firmware files are distributed in the hope that they will be
      useful, but WITHOUT ANY WARRANTY; without even the implied warranty
      of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
  - score: '100.0'
    start_line: 117
    end_line: 119
    matcher: 1-hash
    rule_length: 23
    matched_length: 23
    match_coverage: '100.0'
    rule_relevance: 100
    identifier: other-permissive_66.RULE
    license_expression: other-permissive
    is_license_text: yes
    is_license_notice: no
    is_license_reference: no
    is_license_tag: no
    is_license_intro: no
    matched_text: |
      Permission is hereby granted for the distribution of this firmware data
      in hexadecimal or equivalent format, provided this copyright notice is
      accompanying it.
